ToopherSecondFactor/toopher-openam.js

```diff
@@ -1,183 +1,84 @@
 // vim: ts=2 sw=2 expandtab cindent
```

Comment:
It's worth noting that some arbitrary person on the internet declares "[Modelines are evil](http://damien.lespiau.name/blog/2009/03/18/per-project-vimrc/comment-page-1/)".  Oh, and they also suggest an alternative.